scripts/vmbuild.rb

```diff
@@ -146,7 +147,21 @@ def verify_run(output)
     FileUtils.mkdir_p(destination_directory)
     file_name = "#{name}-#{target}-#{build_label}-#{timestamp}-#{manageiq_checkout.commit_sha}.#{target.file_extension}"
     destination = destination_directory.join(file_name)
-    $log.info `mv #{source} #{destination}`
+
+    case compression
+    when 'gzip'
+      $log.info `gzip -c #{source} > #{destination}.gz`
+      destination = destination.sub_ext(destination.extname + '.gz')
+    when 'zip'
+      $log.info `ln #{source} #{destination}`
```

Comment:
Hard-linking between `source` and `destination` will not always work. `source` is a temporary location and `destination` is where the final images are kept, which could be on different device (and that is the case for our build machine).